lib/facter/framework/config/config_reader.rb

```diff
@@ -33,6 +35,9 @@ def fact_groups
 
       def refresh_config(config_path)
         @conf = File.readable?(config_path) ? Hocon.load(config_path) : {}
+      rescue StandardError => e
+        @log.warn("Facter failed to read config file #{config_path} with the following error: #{e}")
```

Comment:
Is the exception printed out properly? Wouldn't it be easier to read **e.message**?